model/src/main/proto/arguments.proto

```diff
@@ -29,3 +29,22 @@ enum HighlightItem {
   // Indicates that the Developer Options item needs to be highlighted.
   DEVELOPER_OPTIONS_ITEM = 4;
 }
+
+// Arguments needed to open the policy page.
+message PoliciesArguments {
```

Comment:
Suggest: PolicyActivityParams (we use the term 'params' for activities and 'arguments' for fragments).

Reply:
Can we keep it generic as it uses common params/args for both Activity and Fragments.

Reply:
It was separate earlier, @vinitamurthi suggested to keep generic. But as you said it may change over time to made it separate for activity and fragments.